build.gradle.kts

```diff
@@ -32,9 +34,18 @@ apply(from = rootProject.file("dependencies.gradle.kts"))
 apply(from = "scripts/git-hooks/install.gradle")
 
 measureBuilds {
-    enable = project.extra.get("measureBuildsEnabled")?.toString().toBoolean()
-    automatticProject = MeasureBuildsExtension.AutomatticProject.PocketCasts
-    authToken = project.extra.get("appsMetricsToken")?.toString()
+    enable = project.extra.properties.get("measureBuildsEnabled")?.toString().toBoolean()
+    onBuildMetricsReadyListener {
+        val report = this@onBuildMetricsReadyListener
+        SlowSlowTasksMetricsReporter.report(report)
+        runBlocking {
+            InternalA8cCiReporter.report(
+                report,
+                projectName = "pocketcasts",
+                project.extra.get("appsMetricsToken").toString(),
```

Comment:
Minor (🔍): When using named arguments, I usually either use it on all of the parameters or non. Why did you chose to add it for the middle `projectName` parameter, but not for the rest? 🤔

Reply:
I've added named arguments in b3987ad602503b49be84367f12bbc73faa91daca, thanks!

Reply:
Thanks Wojtek! 🙇